src/Domain/Package/PackageManager.php

```diff
@@ -190,19 +190,20 @@ private function initializePackages(FixturePathHandler $fixture_path_handler, st
         continue;
       }
 
-      if ((array_key_exists('version', $datum) || array_key_exists('version_dev', $datum)) && $this->containsValidVersion($datum)) {
-        $this->addPackage($datum, $fixture_path_handler, $package_name);
-        continue;
-      }
-
-      if (isset($datum['core_matrix'])) {
+      if (array_key_exists('core_matrix', $datum)) {
         $constraints = array_values($datum['core_matrix']);
```

Comment:
What if `$datum['core_matrix']` isn't an array? Maybe we should throw an exception with an actionable error message.

Reply:
If `$datum['core_matrix']` isn't an array, the OptionsResolver is will throw an error pointing out that is should be an array and stops tests execution. Are you suggesting that we should show a message that this configuration is incorrect and remove the package from list of packages and continue the execution?

Reply:
Ah. No, I'm satisfied that the `OptionsResolved` will throw an error. I didn't think of that.